rosdep/base.yaml

```diff
@@ -2177,6 +2177,24 @@ libopenscenegraph:
     portage:
       packages: [dev-games/openscenegraph]
   ubuntu: [openscenegraph, libopenscenegraph-dev]
+libopenvdb-dev:
+  arch: [openvdb]
+  debian: [libopenvdb-dev]
+  gentoo:
+    portage:
+      packages: [media-gfx/openvdb]
+  ubuntu: [libopenvdb-dev]
+libopenvdb:
+  arch: [openvdb]
+  debian:
+    jessie: [libopenvdb2.3]
+    sid: [libopenvdb3.2]
```

Comment:
Please define `stretch` but not `sid` stretch is in the process of stabilizing, but sid is a rolling release so we can't target it efficiently.